ui-src/src/app/automated-ui-services.ts

```diff
@@ -16,14 +16,14 @@ export class AutomatedUIServices {
     headers = headers.set("Content-Type", "application/json;");
     headers = headers.append("configPathDir", configPath);
```

Comment:
Can you talk to @angerooney11 about the headers as well as there is duplication happening on the services file.